### ISSUE TITLE
IDEA Plugin - JSTestDriver instance not shared across windows and raises IDE Fatal Error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#185]
```
What steps will reproduce the problem?
1. Open first project in WebStorm/IDEA
2. Start JSTestDriver Server
3. Open second project in WebStorm/IDEA in new window
4. Click on JSTestDriver Server tab. Server us shown as stopped (even though 
its running in 1st window/project)
5. Click on start server. Get an IDE Fatal Error - error starting the server on 
9876.

I think the server instance should be shared across windows (since they are in 
the same EXE instance).

I'll add even if they are not, a nicer error than an IDE Fatal Error should 
probably be raised.

JSTestDriver 1.2.2
JSTestDriver Plugin 1.2.2b
WebStorm WS-102.18

What version of the product are you using? On what operating system?


Please provide any additional information below.

```

Original issue reported on code.google.com by `coolr...@gmail.com` on 5 Jan 2011 at 4:54

**Comment by GoogleCodeExporter** [comment]
```
A current workaround seems to be using "At Address: http://localhost:9876" 
rather than "Running in IDE" for the Server.

The plugin server instance can still be started manually using the JSTestDriver 
Server tab in one of the windows, and you run tests in any IDEA window - they 
use it "remotely". This seems to work pretty well.
```

Original comment by `coolr...@gmail.com` on 5 Jan 2011 at 5:13

**Comment by GoogleCodeExporter** [comment]
Original comment by `corbinrs...@gmail.com` on 5 Jan 2011 at 9:34

**Comment by GoogleCodeExporter** [comment]
```
Yup, there's a fixed port used for communication between JSTD and IDEA as well, 
so you won't be able to run tests in the second IDE window while they're 
running in the first, either.

Maybe I could detect that there's a server running on the default port, and 
indicate that in the UI.
```

Original comment by `aeagle22206` on 5 Jan 2011 at 11:01

**Comment by GoogleCodeExporter** [comment]
```
Note that both windows are running in the same JVM, so there may be some way of 
the server state simply being shared across them. I don't know enough about 
IDEA plugins to confirm that is the case though.
```

Original comment by `coolr...@gmail.com` on 6 Jan 2011 at 3:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 11 May 2011 at 7:38
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 17 May 2011 at 5:37
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3.10 version of the plugin.
```

Original comment by `Sergey.S...@gmail.com` on 23 Nov 2011 at 3:02
- Changed state: **Fixed**
